src/components/VideoPlayer.vue

```diff
@@ -594,6 +594,18 @@ export default {
 
                     const autoDisplayCaptions = this.getPreferenceBoolean("autoDisplayCaptions", false);
                     this.$player.setTextTrackVisibility(autoDisplayCaptions);
+
+                    (() => {
+                        const prefSubtitles = this.getPreferenceString("subtitles", "");
+                        if (prefSubtitles === "") return;
+
+                        const textTracks = this.$player.getTextTracks();
+                        const subtitleIdx = textTracks.findIndex(textTrack => textTrack.language == prefSubtitles);
+                        if (subtitleIdx == -1) return;
```

Comment:
Instead of creating a function and returning, can we just use `if (subtitleIdx != -1) {}`?